Source/WebCore/platform/gstreamer/VideoEncoderPrivateGStreamer.cpp

```diff
@@ -359,6 +360,8 @@ static bool videoEncoderSetEncoder(WebKitVideoEncoder* self, EncoderId encoderId
     }
 
     encoderDefinition->setupEncoder(self);
+    g_object_set(priv->encoder.get(), "qos", TRUE, nullptr);
```

Comment:
Do all encoders have the qos property. Wouldn't be a good idea to check for it?

Reply:
Well it is in the base encoder class and none of the encoders we wrap currently inherits from a different class.